.azure-pipelines/gpu-benchmark.yml

```diff
@@ -28,13 +28,18 @@ jobs:
     cancelTimeoutInMinutes: "2"
     pool: azure-gpus-spot
     container:
-      # TODO: Unpin sha256
-      image: "pytorchlightning/pytorch_lightning:base-cuda-py3.7-torch1.8@sha256:b75de74d4c7c820f442f246be8500c93f8b5797b84aa8531847e5fb317ed3dda"
+      image: "pytorchlightning/pytorch_lightning:base-cuda-py3.7-torch1.8"
       options: "--runtime=nvidia -e NVIDIA_VISIBLE_DEVICES=all --shm-size=32g"
     workspace:
       clean: all
 
     steps:
+      - bash: |
+          # TODO: Prepare a docker image with 1.8.2 (LTS) installed and remove manual installation.
```

Comment:
let's link this with adding tests for PT 1.11

Reply:
You mean I should revert it here and do it in another PR adding 1.11 testing? If so, SGTM and will address it.

Reply:
lest keep this as it is and create new PR by adding an additional PT version, see https://github.com/PyTorchLightning/metrics/pull/1013

Reply:
see: #12984